Library/Homebrew/cask/audit.rb

```diff
@@ -195,10 +195,6 @@ def audit_required_stanzas
 
     sig { void }
     def audit_description
-      # Fonts seldom benefit from descriptions and requiring them disproportionately
-      # increases the maintenance burden.
-      return if cask.tap == "homebrew/cask-fonts"
```

Comment:
This looks like the cause of the test failure. May need to adjust to have it check if the token begins with `font-`.


Will also need to update https://github.com/Homebrew/brew/blob/1fac68348a61f735066ba58e6ba35d2cec8bdf8c/Library/Homebrew/test/cask/audit_spec.rb#L1056-L1063 so that it doesn't rely on `cask.tap` being called here; perhaps change to `expect(cask).to receive(:desc).and_raise(StandardError.new)` to make `cask.desc` fail instead.